docs/examples/DataSet/Accessing-data-in-DataSet.ipynb

```diff
@@ -0,0 +1,854 @@
q++JyLU6WKWFwsEHRcsXjw3TY7o27s7QdX04+0+7wrTCoUui5QdmhM5ZdZGVdho64w5v9avWB2laMj9IG3p+hnNfRNdpW+K6Nq6LREHqC50b6xYtjtc1K6yroSOsq3FXfKmh+ifagzSrDxd9Hpoe/2n3RepvDU9J45qMCEh1EUe6Iw6L1xW5rhqK6Bpx4gOofzK8r40t4TM02Bp37huM+JLWPTXO3Uc+puE4juPkxo2G4ziOk4/JvxhhHtxoOI7jVAIDxrDs+WTFjYbjOE6l8JaG4ziOk4/KLyMyEXGj4TiOUwkMzP00HMdxnNy4R7jjOI6TGx/TGF+UxHS9kiQUbDdwkZk9GMl3BzAfGA7R9ioz2zLiyQ0KfXvfwOL20DGrbijuMKZi6NxnMR+ijGekbk8kct7WMPRdoTfuBEZdeGssw7mv0Bs2iYe2RepqjkdYUzF07ovpqoyWt/Z0hvXHdO2P6xrTyyK+dYWeeIS3wa3bgrT6OZHQgxYJ0Udc17pIND4AIk6LxZ7wXtcNhA53SV2hYlFdu+LOfYPbI9e165BIzkiIPjJ0jVxW25kRkrIQccQcijv3RX8vsVN2xqMExu5roXtBWI/iz7XVhfew/J1QUcx89lQVOBs4Kt1OBb6Y/h/jrWa2IuOY4zhO7fGWxrhzHnBdGkP8HkmzJM03s3ANDMdxnAmNYRk9F1OJjA6PqrEAWFey356mxfi6pJWS/jHt1gqQdLGkFZJWDPaFXSaO4zjjxvDS6Hm2SUytjUbs5R+7om81sxcCL023C2MnM7PlZrbMzJbVN7VWUEzHcZwcWDHfNomputGQdEnaYlgJbABKRwwXpml7YWbr0/87gG8Bp1RDVsdxnLwYYEXLtU1mqm40zOwqMzvezI4HbgLeroTTgN3l4xmS6iXNTf9uAM4FHqm23I7jOCNidkC0NGo9EH4LyXTbVSRTbt8xfEDSytSwNAG3pgajAPwc+EoNZHUcxxmRA2EgXDZFp4hJ2gqsTXfnAuGk76mH6zn1OFB0rbWeS8xs3v6cQNJPSPTIwzYzO2t/6qsVU9ZolCJphZktq7Uc443rOfU4UHQ9UPScCtR69pTjOI4ziXCj4TiO4+TmQDEay2stQJVwPaceB4quB4qek54DYkzDcRzHqQwHSkvDcRzHqQBuNBzHcZzcHBBGQ9LJkoYkvbHWsowXkt4q6eF0u1vSi2ot03gg6SxJj0taJemyWsszHkhaJOl2SY9JelTS+2ot03giqSDpN5J+VGtZnNGZ8kZDUgH4DHBrrWUZZ54CzjCz44BPMgUHFtN7eRVJHJZjgQskHVtbqcaFQeBDZnYMcBpwyRTVc5j3AY/VWggnH1PeaADvBb4HjBzpb5JjZneb2XBowntIFn+capwCrDKz1WbWD9xAEpNlSmFmG4cjWKaLdD5GdsiASY2khcCrga/WWhYnH1PaaEhaAPwZ8KVay1Jl3gn8uNZCjANjib8yJZC0FDgBuLe2kowb/wH8HTC5V/E7gJjSRoPkgfywmU39VcRSJL2cxGh8uNayjAN5469MCSS1krSS329mYXDySY6kc4EtZvZArWVx8lPrVW4rjqRLgHenuzOBG9JAf3OBcyQNmtlNtZKvkpTpeg6Jjl8Fzjaz7TUTbPxoJ0f8lalAuqrz94Bvmtn3ay3POHE68FpJ5wDNwAxJ/21mb6uxXM4IHDDOfZKuBX5kZjfWWpbxQNJi4BfA283s7lrLMx5Iqgf+APwJsB64H3iLmT1aU8EqTBrO+BvADjN7f63lqQaSzgT+r5mdW2tZnJGZci2NA5jLgTnA1WnLanCqrRpqZoOSLiWZCVcArplqBiPldJKQxr9NI1wCfNTMbqmhTI4DHEAtDcdxHGf/meoD4Y7jOE4FcaPhOI7j5MaNhuM4jpMbNxqO4zhObtxoOI7jOLlxo+HUFEl3SPrTsrT3S7p6hDJLJT2S/n186hw27kj6D0kvk/RxSZ8qO3a8pMfSv38uaXY1ZHKcauNGw6k11wPnl6Wdn6bn4XgSb/hxRdJBwGlmdieJbG8uy3I+8K307/8C/nq8ZXKcWuBGw6k1NwLnSmqCZxboOwy4Swmfk/SIpN9K2utFLakR+ATwZkkrJb1Z0ilpPJHfpP8fneadLuk7abyRb0u6V9Ky9NirJP1a0oOSvpuu+VTOG4GfAJjZ48AuSaeWHH8Tyaq7ADcDF1Tm8jjOxMKNhlNT0jWy7gPOSpPOB75tidfp60laEi8CXgF8TtL8krL9JJ7w3zaz483s28DvgZeZ2QnpsX9Ns/81sLMk3shJAJLmAv8AvMLMTgRWAB+MiHo6ULqw3jMtJEmnAdvN7IlUrp1Ak6Q5+3xhHGeC4kbDmQiUdlGVdk29BLjezIbMbDPwS+DkUc41E/huOuZxBfD8knPdAGBmjwAPp+mnkQR0+lW6ZMdfAEsi550PbC3ZvwF4o6Q64t1pW0haTI4zpfC1p5yJwE3A5yWdCEwbDkBEfCn00fgkcLuZ/Vna1XXHKOcS8DMzG607qYdkJVYAzGydpDXAGcAbgBeX5W9OyzjOlMJbGk7NMbNOkpf7Nez9xX4nyXhFQdI84GUkXVmldABtJfszSVbABbioJP0uknEH0tCpL0zT7wFOl/Sc9Nh0Sc+NiPkY8JyytOtJWjNPmln7cGK6Su2hwJqowo4ziXGj4UwUricZu7ihJO0HJN1ID5Es+/53ZraprNztwLHDA+HAZ4FPSfoVyUq4w1wNzJP0MEmAqoeB3Wa2lcS4XJ8euwd4XkS+/wHOLEv7Lkn31w1l6ScB95jZ4GhKO85kw1e5dQ4IJBWABjPrlXQkcBvw3HQwPe857gLONbNdo+S7ErjZzG7bL6EdZwLiYxrOgcJ04PY0Ip6AvxqLwUj5ELAYGNFoAI+4wXCmKt7ScBzHcXLjYxqO4zhObtxoOI7jOLlxo+E4juPkxo2G4ziOkxs3Go7jOE5u/n8Hc6sS0L+vDwAAAABJRU5ErkJggg==\n",
```

Comment:
Units of time(ks) are a bit confusing.

Reply:
i know. this is out of scope of this PR though.